tests/test_clientsecrets.py

```diff
@@ -32,7 +33,8 @@
 DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
 VALID_FILE = os.path.join(DATA_DIR, 'client_secrets.json')
 INVALID_FILE = os.path.join(DATA_DIR, 'unfilled_client_secrets.json')
-NONEXISTENT_FILE = os.path.join(__file__, '..', 'afilethatisntthere.json')
+NONEXISTENT_FILE = os.path.join(os.path.dirname(__file__),
```

Comment:
Follow the principle of line-wrapping at the coarsest grouping of syntax possible. Here this should be
```
NONEXISTENT_FILE = os.path.join(
    os.path.dirname(__file__), 'afilethatisntthere.json')
```
.